packages/components/src/select/Select.js

```diff
@@ -71,7 +71,7 @@ class MultiSelect extends React.Component {
 	 */
 	componentDidMount() {
 		this.select2 = jQuery( `#${ this.props.id }` );
-		this.select2.select2();
+		this.select2.select2({ width: '100%' });
```

Comment:
This should fix the Travis build
```suggestion
		this.select2.select2( { width: "100%" } );
```